# Builds fine with gcc 3.3.5 and without makeinfo (unless you try to
# build the documentation).

--- configure.ac.orig	Mon Mar 17 16:29:16 2014
+++ configure.ac	Thu May 22 19:53:41 2014
@@ -109,10 +109,10 @@ STEPMAKE_PYTHON(REQUIRED, 2.4, 3.0)
 # __GNUC__, but that macro now unofficially means "the compiler
 # supports the GNU C extensions" -- the intel C compiler and clang
 # both support __GNUC__!
-STEPMAKE_GCC_OR_CLANG(REQUIRED, 3.4)
+STEPMAKE_GCC_OR_CLANG(REQUIRED, 3.3)
 
 STEPMAKE_CXX(REQUIRED)
-STEPMAKE_GXX_OR_CLANG(REQUIRED, 3.4)
+STEPMAKE_GXX_OR_CLANG(REQUIRED, 3.3)
 STEPMAKE_GXXCODEGENBUG
 STEPMAKE_CXXTEMPLATE
 STEPMAKE_STL_DATA_METHOD
@@ -226,7 +226,7 @@ fi
 
 STEPMAKE_PATH_PROG(GHOSTSCRIPT, gs, $DOCUMENTATION_REQUIRED, 8.60)
 
-STEPMAKE_PROGS(MAKEINFO, makeinfo, REQUIRED, 4.11)
+STEPMAKE_PROGS(MAKEINFO, makeinfo, OPTIONAL, 4.11)
 STEPMAKE_PROGS(TEXI2HTML, texi2html, $DOCUMENTATION_REQUIRED, 1.82)
 STEPMAKE_PROGS(DBLATEX, dblatex, $DOCUMENTATION_REQUIRED, 0.1.4)
 STEPMAKE_PROGS(BIBTEX, bibtex, $DOCUMENTATION_REQUIRED)
